src/app/pages/dashboard/services/widget-resources.service.spec.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  createServiceFactory,
+  SpectatorService,
+} from '@ngneat/spectator/jest';
+import { mockGlobalStore } from 'app/core/testing/classes/mock-global-store.service';
+import { mockCall, mockWebSocket } from 'app/core/testing/utils/mock-websocket.utils';
+import { App } from 'app/interfaces/app.interface';
+import { Pool } from 'app/interfaces/pool.interface';
+import { WidgetResourcesService } from 'app/pages/dashboard/services/widget-resources.service';
+
+const pools = [
+  { id: 1, name: 'pool_1' },
+  { id: 2, name: 'pool_2' },
+] as Pool[];
+
+const apps = [
+  { id: '1', name: 'app_1' },
+  { id: '2', name: 'app_2' },
+] as App[];
+
+describe('WidgetResourcesService', () => {
+  let spectator: SpectatorService<WidgetResourcesService>;
+  const createService = createServiceFactory({
+    service: WidgetResourcesService,
+    providers: [
+      mockWebSocket([
+        mockCall('replication.query'),
+        mockCall('rsynctask.query'),
+        mockCall('cloudsync.query'),
+        mockCall('webui.main.dashboard.sys_info'),
+        mockCall('interface.query'),
+        mockCall('update.check_available'),
+      ]),
+    ],
+  });
+
+  beforeEach(() => {
+    mockGlobalStore({
```

Comment:
It would be good to refactor `mockGlobalStore`, so that it's mock like other provider mocks and not a separate global function.